### PR TITLE
add LICENSE and package-lock.json files

### PR DESCRIPTION
**Pull request by Maskedman99**
MIT License has been added to the project, A License badge is added to the Readme file, A redudant code has been removed from package.json,  npm 5 upwards uses package-lock.json and has been added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Fast HTML Parser [![NPM version](https://badge.fury.io/js/fast-html-parser.png)](http://badge.fury.io/js/fast-html-parser) [![Build Status](https://travis-ci.org/ashi009/node-fast-html-parser.svg?branch=master)](https://travis-ci.org/ashi009/node-fast-html-parser)
+# Fast HTML Parser [![NPM version](https://badge.fury.io/js/fast-html-parser.png)](http://badge.fury.io/js/fast-html-parser) [![Build Status](https://travis-ci.org/ashi009/node-fast-html-parser.svg?branch=master)](https://travis-ci.org/ashi009/node-fast-html-parser) ![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)
+
```
Comment (ashi009) [inline README.md]: Please remove this additional new line.

```suggestion
```

File: package.json [hunk 2]
```diff
@@ -23,7 +23,9 @@
   "config": {
     "blanket": {
       "pattern": "index.js",
-      "data-cover-never": ["node_modules"]
+      "data-cover-never": [
```
Comment (ashi009) [inline package.json]: nit: Please revert this file, if unchanged.

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (ashi009) [inline LICENSE.md]: please remove this file to LICENSE.

**Comment by Maskedman99** [comment]
Commited Changes

**Pull request merged**